modules/web/js/ballerina/views/variable-definition-statement-view.js

```diff
@@ -99,6 +101,29 @@ class VariableDefinitionStatementView extends SimpleStatementView {
             this.renderDisplayText(model.getStatementString());// Set display text.
         }
     }
+
+    /**
+     * Validate the node type on focus out of the statement's expression editor
+     * @override
+     */
+    validateNode() {
```

Comment:
Can't we have a generic implementation for the validateNode() method in the super class(ballerina-statement-view.js)? If so we will be able to reuse it in assignment-view.js and variable-definition-statement-view.js